### PR TITLE
Platform API - Orders

### PR DESCRIPTION
**Pull request by damianlegawiec**

### CODE REVIEW

File: api/spec/swagger_helper.rb [hunk 1]
```diff
@@ -105,6 +106,48 @@
...
+              included_tax_total: { type: :number },
+              item_count: { type: :integer },
+              approver_id: { type: :string },
+              approved_at: { type: :string, format: :date_time },
+              confirmation_delivered: { type: :boolean },
+              considered_risky: { type: :boolean },
+              token: { type: :string },
+              canceled_at: { type: :string, format: :date_time },
+              canceler_id: { type: :string },
+              store_id: { type: :string },
```
Comment (MatthewKennedy) [inline api/spec/swagger_helper.rb:144]: Are we locking an order to the store it was created in? If so I'm thinking we can drop` store_id: { type: :string }` here maybe?

**Review by MatthewKennedy — APPROVED** [review]
Other than the concern regarding the `store_id` in the schema for the swagger docs this looks good to me.

File: api/spec/swagger_helper.rb (hunk 1 above)
Comment (damianlegawiec) [reply api/spec/swagger_helper.rb:144]: @MatthewKennedy so this schema is also used for update action, and we would like to allow app developers later to re-assign the Order to a different Store

Comment (MatthewKennedy) [reply api/spec/swagger_helper.rb:144]: No problem, all good.

**Pull request merged**